typhos/tests/conftest.py

```diff
@@ -68,6 +72,141 @@ def noapp(monkeypatch):
     )
 
 
+def get_top_level_widgets() -> List[weakref.ReferenceType[QtWidgets.QWidget]]:
+    app = QtWidgets.QApplication.instance()
+    if app is None:
+        return []
+    return [weakref.ref(widget) for widget in app.topLevelWidgets()]
+
+
+def _dump_widgets(widgets: List[weakref.ReferenceType[QtWidgets.QWidget]]) -> None:
+    if not widgets:
+        return
+
+    for widget in widgets:
+        widget = widget()
+        if widget is None:
+            continue
+
+        try:
+            widget.isVisible()
+            widget.windowTitle()
```

Comment:
I think calling one may be sufficient and my reasoning here is a bit hand-wavy/anecdotal:
* It seems like simple boolean attributes may not trigger the "is the object still alive check; if not raise RuntimeError to alert the user"
* So I added `windowTitle()` as a second one, since it requires an actual string to Python string conversion under the hood

It'd be good to dig into the details one day, but for today I think double method call is good enough for me...

Reply:
I'm totally good with being doubly-sure.  Just curious if there was some portion of the widget that doesn't get caught by `widget.isVisible()` or something